Packs/ArcusTeam/README.md

```diff
@@ -0,0 +1,7 @@
+### Cortex Xsoar interfaces with ArcusTeam to help identify, analyze, and predict vulnerabilities and security risks on your connected devices. 
+
+## What Does this pack do?
+* Search for devices and their software in ArcusTeam's growing database of thoroughly inspected devices.
+* Retrieve a list of known vulnerabilities and potential zero days vulnerabilities affecting the device & the software.
+
+Use ArcusTeam's discovery and insights to imporve your organization secuirty posture
```

Comment:
Would you like to add a link to your site? Or your sales team?

Reply:
There is a link in line 9